packages/commons/package.json

```diff
@@ -1,22 +1,23 @@
 {
   "dependencies": {
-    "ansi-regex": "5.0.1",
+    "@babel/preset-env": "^7.19.4",
+    "ansi-regex": "6.0.1",
```

Comment:
If this causes problem, let's upgrade it in a different PR to avoid mixing topics (and remove the `babel` config for now)

Reply:
its gone. removed babel everywhere besides react-ui-kit where it is actively in use for building